src/util/cable_runtime_opts_mod.F90

```diff
@@ -5,6 +5,9 @@ MODULE cable_runtime_opts_mod
   ! hydraulic_redistribution parameters _soilsnow module
 REAL :: wiltParam = 0.0, satuParam = 0.0
 
+! should the above be instead:
+!   REAL :: wiltParam=0.5, satuParam=0.8 ! rk4417 - phase2
```

Comment:
These are default values, the parameters can be set in the namelist file. As long as the default are physically correct, the value used is not particularly important.

Reply:
I removed my 2 commented lines and left both values at zero. I am assuming therefore that zero values for default are physically correct (which was in effect my question).